app/src/main/java/io/neurolab/LongtermGraph.java

```diff
@@ -0,0 +1,5 @@
+package io.neurolab;
+
+public class LongtermGraph {
```

Comment:
Is this an empty class?

Reply:
Yes @CloudyPadmal this has been kept empty in the desktop version as well. I guess it needs to be implemented later.